cmd/gardener-extension-provider-kubevirt/app/app.go

```diff
@@ -146,6 +147,9 @@ func NewControllerManagerCommand(ctx context.Context) *cobra.Command {
 			if err := networkv1.AddToScheme(scheme); err != nil {
 				controllercmd.LogErrAndExit(err, "Could not update manager scheme")
 			}
+			if err := cdicorev1alpha1.AddToScheme(scheme); err != nil {
+				controllercmd.LogErrAndExit(err, "Could not update manager scheme")
```

Comment:
I think it would be better to be a bit more explicit here in the error message and add what failed exactly. The line above returns the same error and would be more informative about which one failed.

Reply:
Yes, I agree. But I don't think this belongs to this PR, it's more a refactoring thing that doesn't have much to do with the handling of pre-allocated data volumes. So I would rather do this in a different PR for some general refactoring / improvements.

Reply:
Alright, maybe an issue in the repo to track that.